src/pke/unittest/utils/UnitTestCCParams.h

```diff
@@ -63,6 +63,7 @@ struct UnitTestCCParams {
     double   keySwitchCount; // BFVRNS, 
     double   multiplicationTechnique; // BFVRNS, 
     double   encryptionTechnique; // BFVRNS,
+    double PREMode; // BGVRNS, BFVRNS, CKKSRNS
```

Comment:
is PREMode supposed to be tested for a limited number of unit tests? if yes, then PREMode should be added as an additional test case data to "struct TEST_CASE_UTxxxxxx". if PREMode may be added to any unit test, then ALL unit tests in ALL files should get the value of DFLT for PREMode.